website/docs/r/elastic_tag_rule.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Elastic"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_elastic_tag_rule"
+description: |-
+  Manages an elastic Tag Rule.
+---
+
+# azurerm_elastic_tag_rule
+
+Manages a elastic Tag Rule.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+resource "azurerm_resource_group" "test" {
+  name     = "example-rg"
+  location = "%s"
```

Comment:
Location should be a valid value like `West Europe`

Reply:
updated